openlibrary/macros/StarRatings.html

```diff
@@ -10,27 +10,34 @@
   <meta property="bestRating" content="5"/>
   <input type="hidden" value="$edition_key" name="edition_id"/>
   <input type="hidden" value="true" name="redir"/>
-  <div class="stars">
-    $ NUM_STARS = 5
-    $for star in range(1, NUM_STARS + 1)
-      <input type="submit" value="$star" name="rating"
-        $if rating == star:
-          class="star-$star star-selected"
-        $else:
-          class="star-$star"
-        id="star-$star" />
-      <label class="star star-$star $('star-selected' if rating==star else '')"
-        $ properties = []
-        $if star == rating:
-          $ properties.append("ratingValue")
-        $if star == 1:
-          $ properties.append("worstRating")
-        $elif (star == NUM_STARS):
-          $ properties.append("bestRating")
-        $if properties:
-          property="$(' '.join(properties))"
-        for="star-$star">$star</label>
-    <span></span>
+  <div class="review">
+    <div class="stars">
+      $ NUM_STARS = 5
+      $for star in range(1, NUM_STARS + 1)
+        <input type="submit" value="$star" name="rating"
+          $if rating == star:
+            class="star-$star star-selected"
+          $else:
+            class="star-$star"
+          id="star-$star" />
+        <label class="star star-$star $('star-selected' if rating==star else '')"
+          $ properties = []
+          $if star == rating:
+            $ properties.append("ratingValue")
+          $if star == 1:
+            $ properties.append("worstRating")
+          $elif (star == NUM_STARS):
+            $ properties.append("bestRating")
+          $if properties:
+            property="$(' '.join(properties))"
+          for="star-$star">$star</label>
+      <span></span>
+    </div>
+    <div class="stats-link">
+      <a class="observations-modal-link" href="javascript:;" data-context="{&quot;username&quot;: &quot;mekBot&quot;, &quot;work&quot;: &quot;/works/OL516825W&quot;, &quot;id&quot;: &quot;stats&quot;}">
```

Comment:
Since it hasn't been mentioned, this is listing mek's account? Should it not be doing that? :P Also this shouldn't be manually encoding; do `$json_dumps({'username': 'mekBot', ...})` or something like that. That will encoding the `"`

Reply:
The encoding should be handled in ObservationsModal.html, which should also be updated to display an image link (or maybe redesigned to decouple the link and the modal's HTML).

Reply:
YUP, this was using testing